app/models/devolved_administration_availability.rb

```diff
@@ -4,4 +4,27 @@ class DevolvedAdministrationAvailability
   embedded_in :local_transaction_edition
   field :type, type: String, default: "local_authority_service"
   field :alternative_url, type: String
+
+  validates :type, inclusion: { in: %w[local_authority_service devolved_administration_service unavailable] }
+  validates :alternative_url, presence: true, if: -> { devolved_administration_service? }
+  validate :alternative_url_format
+
+  def alternative_url_format
+    return if alternative_url.blank?
+
+    begin
+      uri = URI.parse(alternative_url)
+    rescue URI::Error => e
+      Rails.logger.debug e.message.to_s
```

Comment:
I don't think you need to log this - I imagine it'd be a bit confusing in the log without any context.

Reply:
👍